src/dbcp/data_mart/projects.py

```diff
@@ -288,7 +274,16 @@ def _get_proprietary_proposed_offshore(engine: sa.engine.Engine) -> pd.DataFrame
         proj.developer,
         proj."capacity_mw",
         date(proj.proposed_completion_year::text || '-01-01') as date_proposed_online,
-        'active' as queue_status,
+        CASE coalesce(proj.construction_status, 'TBD') -- map to ISO queue_status categories
+            WHEN 'Online' THEN 'completed'
+            WHEN 'Canceled' THEN 'withdrawn'
+            WHEN 'Suspended' THEN 'withdrawn'
+            WHEN 'Not started' THEN 'active'
+            WHEN 'Construction underway' THEN 'active'
+            WHEN 'Site assessment underway' THEN 'active'
+            WHEN 'TBD' THEN 'active' -- assume active
+            ELSE NULL -- shouldn't happen
+            END as queue_status,
```

Comment:
Should we do this in dbcp.transform.offshore_wind? Have a queue_status and raw_queue_status column?

Reply:
Oh ya that's a better place for it, thanks. I'll make `queue_status` with a mapping dict and leave `construction_status` as is